### PR TITLE
beta-20161013 release notes and related docs changes

### PR DESCRIPTION
**Pull request by jseldess**
Release notes based on https://github.com/cockroachdb/cockroach/issues/9781.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/745)

### CODE REVIEW

**Comment by tschottdorf** [comment]
Things I've touched LGTM (rest also looks roughly LGTM, but haven't done fact checking).

**Comment by jseldess** [comment]
Thanks, @tschottdorf. One thing I could use help on: Getting the total count of merged PRs. Though we might want to wait until we chose a cut-off commit.

**Comment by tschottdorf** [comment]
Yeah, should be easy enough to get that with `git log --merges --oneline <old_beta_sha>..<new_beta_sha>`. It picks up some of the cross-merges (`master->develop` and the end of the split), but these would be easy enough to subtract if we care (they are merges, after all).

**Comment by a-robinson** [comment]
The bits I'm familiar with LGTM

**Comment by vivekmenezes** [comment]
LGTM for the stuff I know

**Comment by knz** [comment]
nits:

- "THE" in front of CONCAT_AGG is in all caps.
- We already supported history recalls in the cli shell. What changed with #9704 is the case sensitivity.

File: beta-20161013.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+- [`TIMESTAMP`](timestamp.html) values that include a time zone can now omit the minutes field of the time zone offset, for compatibility with PostgreSQL. [#8666](https://github.com/cockroachdb/cockroach/pull/8666)
+- [`INTERVAL`](interval.html) values can now be input in a colon-delimited format (H:M or H:M_S), for compatibility with PostgreSQL. [#8603](https://github.com/cockroachdb/cockroach/pull/8603)
+- The `INT8` type is now supported as an alias for [`INTEGER`](int.html). [#8858](https://github.com/cockroachdb/cockroach/pull/8858)
+- [`INTERVAL`](interval.html), [`TIMESTAMP`](timestamp.html), [`TIMESTAMPTZ`](timestamp.html), and [`DATE`](date.html) values can now be casted to/from more types. [#9731](https://github.com/cockroachdb/cockroach/pull/9731)
+- The window functions `row_number()`, `rank()`, `dense_rank()`, `percent_rank()`, `cume_dist()`, `ntile()`, `lead()`, `lag()`, `first_value()`, `last_value()`, `nth_value()` are now supported. [#8928](https://github.com/cockroachdb/cockroach/pull/8928), [#9321](https://github.com/cockroachdb/cockroach/pull/9321), [#9335](https://github.com/cockroachdb/cockroach/pull/9335)
```
Comment (bdarnell) [inline beta-20161013.md]: I've been inconsistent in capitalization for function names. I used lowercase here and uppercase below for EXP(), ROUND(), etc. Looks like all the function names below should be lowerecased to match our functions & operators page.

File: beta-20161013.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+- The `EXP()` [function](functions-and-operators.html) returns an error if its argument is greater than 1024 instead of performing excessively expensive computation. [#8822](https://github.com/cockroachdb/cockroach/pull/8822)
+- The `ROUND()` [function](functions-and-operators.html) now breaks ties by rounding to the nearest even value (also known as bankers' rounding). It is now faster, and returns an error when given an excessively large number of digits. [#8822](https://github.com/cockroachdb/cockroach/pull/8822)
+- THE `CONCAT_AGG` [aggregate function](functions-and-operators.html#aggregate-functions) is now supported. [#9690](https://github.com/cockroachdb/cockroach/pull/9690)
+- New [date/time functions](functions-and-operators.html#date-and-time-functions), `EXTRACT_DURATION`, `STRPTIME` and `STRFTIME`, are now supported. [#9734](https://github.com/cockroachdb/cockroach/pull/9734), [#9762](https://github.com/cockroachdb/cockroach/pull/9762)
```
Comment (bdarnell) [inline beta-20161013.md]: `strptime` and `strftime` currently have an `experimental_` prefix.

Comment (jseldess) [inline beta-20161013.md]: Done

**Comment by jseldess** [comment]
Thanks, @knz. Made those changes.

**Comment by bdarnell** [comment]
LGTM

**Pull request merged**